doc/Network/overview.dot

```diff
@@ -0,0 +1,24 @@
+digraph network {
+  label="r³ network overview";
+  graph [ bgcolor="#ff000000" fontname="Liberation Sans" fontcolor="#c0c0c0" ];
+  node [ shape=none fontname="Liberation Sans" fontcolor="#c0c0c0" ];
+  edge [fontname="Liberation Sans"
+        fontcolor="#c0c0c0" color="#c0c0c0"
+  ];
+
+  public  [ label="Public (36)\n89.106.211.64/27" ]
+  inet    [ label="Internet" ]
+  guests  [ label="Guests (127)" ]
+  members [ label="Members (128)\n89.106.211.32/27" ]
+  mgmt    [ label="Management (23)" ]
```

Comment:
again: 23 -> 32
at least this is the case right now... we could change it of course... Is this a proposal to change it or is it a typo?

Reply:
Typo (or rather, I wrote it the wrong way around when I documented it on paper)